sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/randomExpressions.scala

```diff
@@ -77,18 +86,28 @@ case class Rand(seed: Long) extends RDG {
   }
 }
 
+object Rand {
+  def apply(seed: Long): Rand = Rand(Literal(seed))
+}
+
 /** Generate a random column with i.i.d. gaussian random distribution. */
+// scalastyle:off line.size.limit
 @ExpressionDescription(
-  usage = "_FUNC_(a) - Returns a random column with i.i.d. gaussian random distribution.")
-case class Randn(seed: Long) extends RDG {
-  override protected def evalInternal(input: InternalRow): Double = rng.nextGaussian()
+  usage =
+    """
+      _FUNC_() - Returns a random column with i.i.d. gaussian random distribution.
```

Comment:
Nit: gaussian -> Gaussian. Well, really we need to specify that it has mean 0 and stdev 1. And that's called the standard normal distribution. And I assume the 'n' in 'randn' is for normal. So maybe better to say "Returns a random column with i.i.d. values drawn from the standard normal distribution" everywhere including the comment.